package.json

```diff
@@ -69,6 +69,7 @@
         "react-top-loading-bar": "^1.2.0",
         "redux": "^4.0.4",
         "redux-thunk": "^2.3.0",
+        "ts-morph": "^25.0.0",
```

Comment:
We can add this under `devDependencies` i believe since we don't use this in the runtime.

Reply:
Yes, we can

Reply:
Fixed with [d94f42f](https://github.com/wso2/identity-apps/pull/7329/commits/d94f42fc77438733b86382feb9f7f26a537c7ed9)